### DISCUSSION TITLE
Update to .NET 8.0 Desktop Runtime

### DISCUSSION THREAD

**Discussion by StefanH1303** [seerge/g-helper#1696] [Ideas]
Hello,
I am using PowerToys, FanControl and GHelper on my system. The other two recently upgraded to .NET 8.0. So I wanted to ask, if there is already a plan to also upgrade GHelper to 8.0? Would be really great as I can then uninstall 7.0 from my system :)

Thanks and keep up the good job!

**Comment by seerge** [comment]
@StefanH1303 hello, I'm not a big fan of that atm, cause it will force existing app users to install .NET8 (which they don't have installed most probably) :)

**Reply by StefanH1303** [reply]
Sure, but there are probably also a lot of people which never install any dotnet runtime until it is needed. I cannot tell you how much effort this would be, but FanControl offers currently versions for 4.8, 7.0 and 8.0 as different downloads. Perhaps that would be a way to go and even measure how much which version is downloaded? (No need to start with 4.8 now though 😅)

**Comment by ke1ne** [comment]
@StefanH1303 God, please, no, I have already different .nets for different apps.

**Reply by StefanH1303** [reply]
Perhaps a third option would be to include the required runtime libraries into the download like Power Toys does? Also I have no idea what that implies in regards to overhead...

**Reply by ke1ne** [reply]
I would simply wait at least some months, when 8.0 will spread.

**Comment by borisblagov** [comment]
Hello. I just tried to start Ghelper which worked until yesterday without problems and is now telling me that this app needs .net 8.0. Did something change? I didn't even update it and as far as I know you don't update it automatically.

Thank you for your work.

**Reply by seerge** [reply]
@borisblagov hello, app uses .NET8 (instead of .NET7) starting from [build 0.179](https://github.com/seerge/g-helper/releases/tag/v0.179) released **2 month ago**.  Most likely you didn't update it for a long time, and now updated it.

Just download latest .NET8 and you will be fine

**Reply by borisblagov** [reply]
You are correct on both terms, I rarely update apps :D. I will, can't image my laptop without the G-helper or with the Armory Bloatware. Thanks!
